deploy.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-ember build -e production
+ember build
```

Comment:
why this change? Seems that we would want to build for production when deploying.